### DISCUSSION TITLE
Make the base branch dropdown search case insensitive

### DISCUSSION THREAD

**Discussion by snemvalts** [community/community#11904] [Pull Requests]
The branch search button for selecting a base branch is case sensitive, so capitalization matters in the search box. 
Note the difference between using "core" and "CORE" as a search term: 

![Screenshot 2022-02-22 at 14 56 54](https://user-images.githubusercontent.com/3755903/155137136-f09309a9-94f8-4e78-a7c1-3e34609e9573.png)
![Screenshot 2022-02-22 at 14 56 57](https://user-images.githubusercontent.com/3755903/155137141-e546881e-2f00-4b52-a479-5cd1e90309bb.png)

I would argue that case insensitive search is the default used in all web applications, making it insensitive here as well would: 
* make search clearer, remove gotchas
* align better with preexisting notions users have with search (don't need to worry about case in the term).

**Comment by Kevin-Andrew** [comment]
I agree with @snemvalts. For some other examples:
![image](https://user-images.githubusercontent.com/22459191/222209962-0ae4e4c4-ebad-46c9-aacf-da1863e5214c.png)

![image](https://user-images.githubusercontent.com/22459191/222209839-cad3f7cc-46c5-473e-93ea-f920b5ee8bc5.png)

Note that when searching for a file within the repository using the "Go to file", the search is case insensitive.
![image](https://user-images.githubusercontent.com/22459191/222210393-1fce896e-f869-4dea-99f1-48a08aa887c0.png)

So even more to @snemvalts point, one could argue that the search functionality within GitHub.com itself is inconsistent. Forget about the rest of the web, the number one priority is to at least be consistent within your own application.

If you do feel the need to support case sensitive searches, then what you might be able to do is what Microsoft's Visual Studio has done in parts of its UI. That is, if a user enters a mixed case search terms, then chances are, they want a case sensitive search. Otherwise, do a case insensitive search.

**Comment by intelyfabs** [comment]
This would be an excellent feature to have. Not only does it make plain sense but it would improve UX by a significant factor
